R/vector_control_parameters.R

```diff
@@ -133,3 +133,59 @@ set_spraying <- function(
   parameters$spraying_ms_gamma <- ms_gamma
   parameters
 }
+
+#' @title Parameterise a habitat management strategy
+#'
+#' @description The model will simulate larval source habitat management at 
+#' `timesteps` to the entire human population. 
+#' The impact will reduce adult mosquitoes recruited from pupae to the susceptible
+#' cohort.
+#'
+#' The rapidity of the impact is captured by the lsm_factor function
+#' by changing the parameters lsm_rate and deprec_param
+#'
+#' The amount of impact in terms of reduced adult mosquito densities
+#' that the habitat management has is determined by larvi_min which is
+#' set to 1 as default when there is no impact.
+#'
+#' Habitat management effects will be randomly removed each timestep with a rate of `1 -
+#' exp(-1/habitat_management_waning)`
+#'
+#' The structure for the habitat model is documented in the
+#' working draft Sherrard-Smith et al (larviciding in Kenya)
+#'
+#' @param parameters a list of parameters to modify
+#' @param timesteps the timesteps at which to distribute lsm rounds
+#' @param larvi_min the proportion of the mosquito population reduced (0 to 1), 1 is no impact
+#' @param lsm_rate the duration til the impact is acheived
```

Comment:
is this the time it takes for the intervention to reach its full effect?

Reply:
i have changed the descriptions to hopefully clarify things